### ISSUE TITLE
Show same context menu options as the native explorer.

### ISSUE THREAD

**Issue by cristianCanek** [derceg/explorerplusplus#210]
I normally use git bash from windows to manage my changes in my projects, Explorer++ is a great solution for the issue of having a lot of explorer windows opened, one of the most important pending options is to have the same options in the context menu (right click) as the native explorer has. It would be great to have it specially for those persons that uses many different programs and they normally are integrated in the context menu.

**Comment by yhyu13** [comment]
@derceg I have the exact same problem as @cristianCanek. Does your commit solve this problem? How can enbale this feature? I am using v1.3.5 (stable).

**Comment by derceg** [comment]
@yhyu13 Yes, it should. You'll need to use the one of the recent development builds on Windows 8+.

**Comment by derceg** [comment]
In Windows 8+, the context menu should be the same, apart from some minor differences (e.g. the undo/redo items are removed, since they're non-functional). In Windows 7, it's not feasible to use the original context menu, so there's nothing that can be done there. So this functionality is effectively fully implemented.
